client/my-sites/theme/main.jsx

```diff
@@ -1111,9 +1100,10 @@ class ThemeSheet extends Component {
 						{ translate( 'Activate this design' ) }
 					</span>
 				);
+			} else if ( defaultOption.label === translate( 'Activate' ) ) {
+				return translate( 'Activate this design' );
```

Comment:
See below comment for context: https://github.com/Automattic/wp-calypso/pull/94505/files#r1757398420